### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.cognitiveservices.vision.face.models.api_error.APIError
  fullName: azure.cognitiveservices.vision.face.models.api_error.APIError
  inheritance:
  - inheritance:
    - type: builtins.object
    type: msrest.serialization.Model
  langs:
  - python
  module: azure.cognitiveservices.vision.face.models.api_error
  name: APIError
  source:
    id: APIError
    path: azure-cognitiveservices-vision-face\azure\cognitiveservices\vision\face\models\api_error.py
    remote:
      branch: master
      path: azure-cognitiveservices-vision-face\azure\cognitiveservices\vision\face\models\api_error.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 16
  summary: Error information returned by the API.
  syntax:
    content: APIError(**kwargs)
    parameters:
    - description: ''
      id: error
      type:
      - azure.cognitiveservices.vision.face.models.Error
  type: class
  uid: azure.cognitiveservices.vision.face.models.api_error.APIError
references: []
